management/server/rule.go

```diff
@@ -0,0 +1,97 @@
+package server
+
+import (
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// TrafficFlowType defines allowed direction of the traffic in the rule
+type TrafficFlowType int
+
+const (
+	// TrafficFlowBidirect allows traffic to both direction
+	TrafficFlowBidirect TrafficFlowType = iota
+)
+
+// Rule of ACL for groups
+type Rule struct {
```

Comment:
Maybe we should call it ACL? 
And all the corresponding go files, handlers rename to ACLS.
Similarly for the http endpoint it might be better to name things as ACL.

What do u think?
@gigovich @mlsmaycon

Reply:
IMHO: Rule is a part of the ACL.

Reply:
So as agreed we just rename a Handler to Acls